lib/chewy/type/crutch.rb

```diff
@@ -23,7 +24,11 @@ def #{name}
 
       module ClassMethods
         def crutch name, &block
-          self._crutches = _crutches.merge(name.to_sym => block)
+          if block
+            self._crutches = _crutches.merge(name.to_sym => block)
+          else
+            self._crutches_classes = _crutches_classes + [name]
```

Comment:
Aren't crutches memoized? We will have one instance per batch. his change would allow simplify crutches definition I guess.

additionaly If we could support something like:
```
crutch CrutchClass
...
field :name, crutch: [:crutch_class, :crutch_method] # translated to value: -> (item, crutch) {crutch.crutch_class.crutch_method(item) }
```